.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [develop]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v7
```

Comment:
One i thing i realised is that we can add some artifact cache here

```
      - name: cache SBT
        uses: coursier/cache-action@v3
```

Reply:
This job only runs the releases, which I think is desirable to run with a clean machine. My experience is that GitHub Actions is usually fast enough for most projects without caching

Reply:
right, anecdotally I've seen it fail a few times ¯\_(ツ)_/¯ I guess we can always re-trigger?

Reply:
You can always rerun the job or force push the git tag

Reply:
right that's what I meant. I understand that this is probably a preference thing but i don't feel that fetching artifacts every time offer a lot of guarantees. If a build is green on `develop` for a given hash, running the release job (`develop` or `tag`) should mean that it will equally build and not be subjective to transient errors. Publishing to sonatype should be the only thing that can fail (which fails quite often unfortunately)